src/Input.test.jsx

```diff
@@ -24,3 +19,19 @@ test('Input', () => {
 
   expect(handleClick).toBeCalled();
 });
+
+test('Change Event', () => {
+  const handleChange = jest.fn();
+
+  const { getByPlaceholderText } = render((
+    <Input
+      onChange={handleChange}
+    />
+  ));
+
+  expect(handleChange).not.toBeCalled();
+
+  fireEvent.change(getByPlaceholderText('할 일을 입력해 주세요'), { target: { value: '뿌잉' } });
+
+  expect(getByPlaceholderText('할 일을 입력해 주세요')).toHaveAttribute('value', '뿌잉');
```

Comment:
value 예상값 '뿌잉'을 넣었는데, 왜 빈 값이 들어오는 걸까요? ㅜㅜ

Reply:
해당 컴포넌트에서는 value 를 테스트하기 보다, handleChange 가 적절하게 불리어졌나? 를 체크하는게 더 맞는거같아요 :)
(value는 props 로 받기때문에 안들어갔을수도 있다고 생각합니다..!)
```suggestion
  fireEvent.change(getByPlaceholderText('할 일을 입력해 주세요'), { target: { value: '뿌잉' } });

  expect(onChange).toHaveBeenCalledWith({taget: { value: '뿌잉' } }) 
```

Reply:
고쳐주신부분만 수정하면 테스트 페일드 뜹니다 ㅠ_-

`onChange`는 변수인 걸까요?
제가 `input`이라는 변수에는 `getByPlaceholderText('할 일을 입력해 주세요')`를 담았는데요,
`onChange`에는 어떻게 담아야할까요?

Reply:
제가 오해할만하게 리뷰를 적은것 같습니다.  onChange 는 밑에 기준으로 `handleChange` 를 의미하는것 이였습니다.. 😱 관련해서는 제 실수로인해 생긴 에러이므로, 이부분만 급하게 코드리뷰를 진행하고, 내일 아침에 다시 코드리뷰를 진행하겠습니다 ;)
```jsx
  const { getByPlaceholderText } = render((
    <Input
      onChange={handleChange}
    />
  ));
```